django/contrib/auth/models.py

```diff
@@ -377,8 +377,8 @@ def get_short_name(self):
         return self.first_name
 
     def email_user(self, subject, message, from_email=None, **kwargs):
-        """Send an email to this user."""
-        send_mail(subject, message, from_email, [self.email], **kwargs)
+        """Send an email to this user and return the number of sent emails."""
```

Comment:
My spellcheck tool of choice (languagetool) suggests a little change.
```suggestion
        """Email this user and return the number of emails sent."""
```